mill/image_resize.go

```diff
@@ -133,8 +133,15 @@ func encodeImage(reader io.Reader, format Format, width int, quality int) (*byte
 		}
 
 		resized := imaging.Resize(img, width, 0, imaging.Lanczos)
-		if err = jpeg.Encode(buff, resized, &jpeg.Options{Quality: quality}); err != nil {
-			return nil, nil, err
+
+		if format == PNG {
```

Comment:
quick fix to make sure we encode pngs correctly

Reply:
sweet